packages/sortable-behavior-bundle/tests/Functional/AbstractSortableAdminTest.php

```diff
@@ -32,15 +38,31 @@ public function itUpdatesPosition(): void
         $client = static::createClient();
 
         /**
-         * @phpstan-var AnonymousFactory<SortableEntity>
+         * @psalm-suppress DeprecatedClass
+         *
+         * @todo: simplify when dropping support for ZenstruckFoundryBundle < 1.10
          */
-        $factory = AnonymousFactory::new(SortableEntity::class);
+        if (class_exists(AnonymousFactory::class)) {
```

Comment:
To avoid deprecations, you should use inverse logic, If the new function exist I use it, otherwise fallback to the old logic.